src/Emmet/Parser/InputElement.purs

```diff
@@ -0,0 +1,55 @@
+module Emmet.Parser.InputElement where
+
+import Prelude
+import Emmet.Parser.Data (EmmetParser)
+import Control.Alt ((<|>))
+import DOM.HTML.Indexed.InputType (InputType(..)) as IT
+import Data.List (many)
+import Data.Newtype (wrap)
+import Emmet.Parser.Element (parseElementName, parseClass, parseId, parseGeneralStringAttribute)
+import Emmet.Types (Emmet, element)
+import Emmet.Attribute(Attribute(..), InputType)
+import Text.Parsing.Parser (fail)
+import Text.Parsing.Parser.Combinators (try)
+import Text.Parsing.Parser.Combinators as P
+import Text.Parsing.Parser.String (char, oneOf, string)
+
+inputType :: EmmetParser InputType
+inputType = wrap <$> P.choice [
```

Comment:
All of these don't need parens, I'd prefer the constructor name instead of `wrap`.